CIPs/cip-granda_mento.md

```diff
@@ -0,0 +1,188 @@
+3. Create a large limit order of `fiat <-> stable token` on a centralized exchange in hope it gets filled over time.
+
+The existing implementation of Mento is suitable for facilitating low volume exchanges, e.g. up to thousands of stable tokens at a time, with minimal slippage. For cUSD, slippage of over 2% starts to occur on both Mento and on centralized exchanges at trade sizes of ~$50k+. OTC trading satisfies the needs of medium volume exchanges, e.g. ~$100k, but similarly takes a hit of around 2-3%. For lower total supply stable tokens with lower liquidity, larger OTC trades or centralized exchange orders may not be possible.
+
+Increasing Mento bucket sizes permanently to allow larger exchanges via Mento will reduce slippage, but it puts a more significant portion of the Reserve at risk.
+
+Building a process to exchange large (~$1 million+) amounts of CELO for stable tokens will enable large entities to make significant block purchases of stable tokens at scale. While minting of the stable token is expected to be the main use case, reversibility (ie exchanging `stable token -> CELO`) is desirable to instill higher confidence when exchanging large quantities.
+
+While Granda Mento will help facilitate `CELO <-> stable token` exchanges, this is ultimately meant as a mechanism to help purchasers with fiat (e.g. USD) who are seeking to own Celo's corresponding stable token (e.g. cUSD). Purchasers would come to an agreement with a broker (e.g. the Celo Foundation) who owns an existing amount of CELO. The broker would be the one to make the `CELO -> cUSD` exchange on-chain.
+
+The proposed implementation is being considered in the medium-term for large scale exchanges until Mento is possibly reworked to support more efficient large-scale minting on a sustained basis.
+
+## Specification
+
+At a high level, the design involves:
+1. Anyone can submit an exchange proposal.
+   * The amount of stable tokens being bought/sold in the exchange is required to be within a governable range.
+   * The assets being sold (ie either the stable token or CELO) in the exchange are deposited.
+   * The current oracle price for the exchange is recorded.
+   * Off-chain, each proposal should be shared with the community in a forum post. For transparency, the post should share where the token being sold is coming from, when it was acquired, and at what price it was acquired.
+2. The proposed exchange must be approved by a multisig that has previously been authorized by Governance. At this point, the exchange still cannot be executed.
+3. A forced waiting period of X days must elapse before the exchange can be executed. During this time, Governance can choose to veto the exchange, refunding the exchange proposer.
+4. After the waiting period, the exchange is executed according to the oracle price recorded in step 1. A fee is imposed upon the exchange through the use of a "spread".
+
+An exchange can have the following states:
```

Comment:
This feels like 2 proposals, one for enabling Granda Mento and another for defining the process around submitting an exchange proposal for the Granda Mento. Also, who will be coordinating those proposals?

Reply:
Around the next contract release when the implementation is deployed, cLabs (probably me) will propose the CGP to activate Granda Mento for cUSD and cEUR, which will probably be a single one-time proposal. As subsequent stabletokens are released, I imagine activating Granda Mento for the new stable token will be included in the process of launching the stable token

As for the process for submitting an exchange proposal, currently I feel like the CIP doesn't give much info on that, but I would expect the exchange proposer would be responsible for some process like a forum post, possibly chatting about it in a governance call, etc.